.pypirc

```diff
@@ -0,0 +1,6 @@
+[distutils]
+index-servers=
+    test
```

Comment:
can you add a `pypi` section too so we can upload to the regular the place the regular way?